eng/spotbugs-aggregate-report/pom.xml

```diff
@@ -112,7 +112,7 @@
         <dependency>
           <groupId>com.microsoft.azure</groupId>
           <artifactId>azure-batch</artifactId>
-          <version>8.1.0</version> <!-- {x-version-update;com.microsoft.azure:azure-batch;current} -->
+          <version>8.1.1</version> <!-- {x-version-update;com.microsoft.azure:azure-batch;current} -->
```

Comment:
I don't see an `8.1.0` version in Maven, are we looking to release this fix as `8.1.0` or `8.1.1`?

Reply:
I think the previous version was 8.1.0-beta.1 (which is a misnomer). The new version should be 8.1.1.

Reply:
@gingi I don't see 8.1.0-beta.1 either in Maven. The last released version is `8.0.0`. So, the next version should be `8.0.1` if it's a patch release or `8.1.0` if it's a minor version release.